sdks/java/core/src/main/java/org/apache/beam/sdk/testing/SourceTestUtils.java

```diff
@@ -38,6 +38,7 @@
 import java.util.concurrent.Executors;
 import java.util.concurrent.Future;
 import javax.annotation.Nullable;
+import org.apache.beam.sdk.coders.CannotProvideCoderException;
```

Comment:
Source.getDefaultOutputCoder is also called from a bunch of other methods in SourceTestUtils. Those methods need to have new overloads where the user provides a Coder explicitly, for the same reason as above.